component_styles/input.css

```diff
@@ -0,0 +1,46 @@
+.input-group {
+    color: #333;
+    font-size: .8rem;
+    line-height: 20px;
+    margin: 0 20px 10px;
+    width: 80%;
+}
+  
+label {
+    display: block;
+    margin-bottom: 2px;
+}
+
+input[type=text] {
```

Comment:
these styles will apply only to inputs of type text and if the user of the component library uses input of type other than text then it might be an issue(if it is done intentionally then it is fine)